modules/static/envoyproxy/protoc-gen-validate/initref

```diff
@@ -1 +1 @@
-845944919806c98c7cfacdf82817524392ef7c83
+v0.5.0
```

Comment:
This is the closest tag:

```
➜  protoc-gen-validate git:(8459449) git describe --all 845944919806c98c7cfacdf82817524392ef7c83
tags/v0.5.0-4-g8459449
```

Here are all the releases: https://github.com/bufbuild/protoc-gen-validate/releases